test/docfx.Test/lib/JsonSchemaTest.cs

```diff
@@ -330,13 +330,13 @@ public void TestJsonSchemaConfirmance(string description, string schemaText, str
             "['warning','invalid-value','Invalid value for 'key1[0]': 'yyy'','file',1,15]")]
 
         // additional errors
-        [InlineData("{'required': ['author'], 'additionalErrors': {'author': {'missing-attribute': {'severity': 'suggestion', 'code': 'author-missing', 'message': 'Add a valid GitHub ID.'}}}}", "{'b': 1}",
+        [InlineData("{'required': ['author'], 'customErrors': {'author': {'missing-attribute': {'severity': 'suggestion', 'code': 'author-missing', 'additionalMessage': 'Add a valid GitHub ID.'}}}}", "{'b': 1}",
```

Comment:
Change docs build too, from 'additionalErrors' to 'customErrors'

Reply:
https://dev.azure.com/ceapex/Engineering/_git/Docs.Build/pullrequest/14134?_a=overview